sfm_pc/management/commands/import_google_doc.py

```diff
@@ -492,12 +554,12 @@ def create_organization(self, org_data):
                     'Organization_OrganizationName': {
                         'value': name_value,
                         'confidence': confidence,
-                        'sources': sources
+                        'sources': sources.copy()
```

Comment:
This fixes the issue that inspired this PR: with sources assigned via shallow copies instead of references, we can comfortably run e.g. data['Organization_OrganizationName']['sources'] += new_sources` without accidentally mutating other dict keys that reference this object.